composer.json

```
@@ -24,6 +24,7 @@
         "symfony/security": "3.3.17 || 3.4.7 || 3.4.8 || 3.4.11",
         "symfony/swiftmailer-bundle": "2.6.* <2.6.2",
         "symfony/twig-bundle": "4.1.0",
+        "symfony/web-profiler-bundle": ">=6.1, <6.1.2",
```

```suggestion
        "symfony/web-profiler-bundle": ">=6.1 <6.1.2",
```
we usually don’t use the comma AFAIK.